src/vcgen.cpp

```diff
@@ -89,9 +89,13 @@ llvm::cl::opt<unsigned int> num_memblocks("num-memory-blocks",
   llvm::cl::init(0), llvm::cl::value_desc("number"),
   llvm::cl::cat(MlirTvCategory));
 
+llvm::cl::opt<unsigned int> max_unknown_dimsize("max-unknown-dimsize",
+  llvm::cl::desc("Maximum dimension size for unknown shaped dimension"
+                    "(default value: 1000)"),
+  llvm::cl::init(50), llvm::cl::value_desc("number"),
```

Comment:
50? 1000..?

Reply:
Updated to 50. Because default value `1000` causes timeout in some cases.